src/test/java/org/shikshalokam/backend/scp/SelfCreationPortalBaseTest.java

```diff
@@ -33,5 +34,21 @@ public static Response loginToScp(String loginId, String password) {
             e.printStackTrace();
         }
         return response;
+    }public static Response loginToScpContentCreator(String loginId, String password) {
```

Comment:
why we need duplicate method?
just before the need login with needed credentials